src/table/table-role/grid-navigation.tsx

```diff
@@ -135,7 +144,13 @@ class GridNavigationHelper {
   };
 
   private onFocusout = () => {
-    this.focusedCell = null;
+    // When focus leaves the cell and the cell becomes no longer belong to the table it indicates the focused element has been unmounted.
```

Comment:
Minor: "When focus leaves the cell and the cell ~becomes~ no longer belongs to the table, it indicates the focused element has been unmounted."